gateway/src/apicast/policy/upstream/apicast-policy.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "http://apicast.io/policy-v1/schema#manifest#",
   "name": "Upstream",
-  "summary": "Allows to modify the host of a request based on its path.",
+  "summary": "Allows to modify the upstream URL of the request based on its path.",
   "description":
-    ["This policy allows to modify the host of a request based on its path. ",
-     "It accepts regular expressions that, when matched against the path, ",
-     "replace it with a given string. \n",
+    ["This policy allows to modify the upstream URL (scheme, host and port) of the request based on its path. ",
```

Comment:
That's to avoid the confusion - we don't only change the `host`, but also `scheme` and `port`